ambari/security/advanced-security/ldap-pdc.rst

```diff
@@ -0,0 +1,216 @@
+#. Set a password when prompted. You will use this during ambari-server setup-ldap.
+
+Then run:
+
+::
+
+  ambari-server setup-ldap
+
+1. At the ``Primary URL*`` prompt, enter the server URL and port you collected above. Prompts marked with an asterisk are required values.
+
+2. At the ``Secondary URL*`` prompt, enter the secondary server URL and port. This value is optional.
+
+3. At the ``Use SSL*`` prompt, enter your selection. If using **LDAPS**, enter ``true``.
+
+4. At the ``User object class*`` prompt, enter the object class that is used for users.
+
+5. At the ``User name attribute*`` prompt, enter your selection. The default value is ``uid``.
+
+6. At the ``Group object class*`` prompt, enter the object class that is used for groups.
+
+7. At the ``Group name attribute*`` prompt, enter the attribute for group name.
+
+8. At the ``Group member attribute*`` prompt, enter the attribute for group membership.
+
+9. At the ``Distinguished name attribute*`` prompt, enter the attribute that is used for the distinguished name.
+
+10. At the ``Base DN*`` prompt, enter your selection.
+
+11. At the ``Referral method*`` prompt, enter to ``follow`` or ``ignore LDAP`` referrals.
+
+12. At the ``Bind anonymously*`` prompt, enter your selection.
+
+13. At the ``Manager DN*`` prompt, enter your selection if you have set **bind.Anonymously** to false
+
+14. At the ``Enter the Manager Password*`` prompt, enter the password for your LDAP manager DN.
+
+15. If you set ``Use SSL*`` = true in step 3, the following prompt appears: ``Do you want to provide custom TrustStore for Ambari?`` Consider the following options and respond as appropriate:
+
+  * **More secure option**: If using a self-signed certificate that you do not want imported to the existing JDK keystore, enter ``y``. For example, you want this certificate used only by Ambari, not by any other applications run by JDK on the same host. If you choose this option, additional prompts appear. Respond to the additional prompts as follows:
+
+    * At the ``TrustStore type`` prompt, enter ``jks``.
+    * At the ``Path to TrustStore file`` prompt, enter ``/keys/ldaps-keystore.jks`` (or the actual path to your keystore file).
+    * At the ``Password for TrustStore`` prompt, enter the password that you defined for the keystore.
+
+  * **Less secure option**: If using a self-signed certificate that you want to import and store in the existing, default JDK keystore, enter ``n``.
+
+    * Convert the SSL certificate to X.509 format, if necessary, by executing the following command (Where ``<slapd.crt>`` is the path to the X.509 certificate.):
+
+      ::
+
+        openssl x509 -in slapd.pem -out <slapd.crt>
+
+    * Import the SSL certificate to the existing keystore, for example the default jre certificates storage, using the following instruction (Where Ambari is set up to use JDK 1.7. Therefore, the certificate must be imported in the JDK 7 keystore.):
```

Comment:
We use JDK1.7 and JDK1.8. Restate sentence using JDK 1.8 (as done in other parts of Ambari doc).